README.md

```diff
@@ -224,6 +224,58 @@ sudo setcap 'cap_net_bind_service=+ep' AuthServer
 ```
 and then go to `build/masterconfig.ini` and change `use_sudo_auth` to 0.
 
+### Linux Service
+If you are running this on a linux based system, it will use your terminal to run the program interactively, preventing you using it for other tasks and requiring it to be open to run the server.  
+_Note: You could use screen or tmux instead for virtual terminals_  
+To run the server non-interactively, we can use a systemctl service by creating the following file:  
+`/etc/systemd/system/darkflame.service`  
+with the contents of:
+```ini
```

Comment:
What would be your thoughts on moving this to an exmaple file, something like systemd.example, and then having users rename/move that file to the correct name instead of having it in the readme as is?  That way they can just copy and rename the file and make the edits for the process.

Reply:
yeah, this would be good

Reply:
Applied changes